tests/ast/data/functions1.test

```diff
@@ -362,9 +362,9 @@ df27 = df.select(bitshiftright("A", col("B")), bitshiftright("A", -10), bitshift
 
 df28 = df.select(bround("A", 10), bround("A", 2), bround(col("A"), col("B")))
 
-df29 = df.select(convert_timezone("A", col("B"), None), convert_timezone(col("A"), "B", None), convert_timezone("A", "B", None), convert_timezone(col("A"), col("B"), None), convert_timezone("A", "B", None), convert_timezone("A", "B", "A"))
```

Comment:
did we add `None` as the last pos arg here as a hack to make things work earlier? Does your refactor fix that?

Reply:
This is actually because we were including a `None` arg in our protobuf message, so the Unparser was generating the `None` here. My refactor fixes this by removing the addition of any arguments that are `None` by default from the protobuf message.